plugin/ochttp/trace.go

```diff
@@ -158,3 +149,30 @@ func responseAttrs(resp *http.Response) []trace.Attribute {
 		trace.Int64Attribute(StatusCodeAttribute, int64(resp.StatusCode)),
 	}
 }
+
+func status(statusCode int) trace.Status {
+	st := trace.Status{}
```

Comment:
This only sets the code. Why not also set the message to the text of the status line?